site/en/docs/privacy-sandbox/fledge-api/interest-groups/index.md

```diff
@@ -464,6 +464,32 @@ function to request the browser remove the interest group.
 
 Code for an ad can also call this function for its interest group.
 
+## Frequently asked questions
+
+{% Details %}
+{% DetailsSummary %}
+### How do you implement frequency control by click?
+{% endDetailsSummary %}
+
+For simple frequency control, you can use the `prevWins` field in `browserSignals` inside `generateBid()`. Another mechanism is to call `navigator.leaveAdInterestGroup()` to request that a user's browser leave an interest group when an ad is clicked. This prevents future bidding and acts as a form of frequency capping. 
+
+You can also use a first-party cookie to store click information. When the ad is rendered, overwrite an existing interest group with the click data as user bidding signals. The workflow would look something like: 
+
+*   User visits `advertiser.com/product`.
+*   The advertiser writes "0 clicks" in a first-party cookie and calls `joinAdInterestGroup({ ..., userBiddingSignals: { clicks: [] } })`.
+*   User clicks on an ad at a later time and is taken to `advertiser.com/product`.
+*   The advertiser reads and increments first-party cookie click data, then calls `joinAdInterestGroup({ userBiddingSignals: { clicks: ["1667499972"] } })`.
+*   For future bidding, the click data available in `userBiddingSignals` can be used in bidding logic.
+{% endDetails %}
+
+{% Details %}
+{% DetailsSummary %}
+### How to update the adComponents by user's browsing history to use for recommendation?
```

Comment:
```suggestion
### How do I use a user's recent browsing history for ad recommendations?

The [`adComponents` property](#specify-ads-for-an-interest-group) can include relevant metadata to inform your bid.
```

Reply:
This is a recommendation -- my assumption is they may not know about `adComponents` but do have the question about browsing history and recommendations.